website/content/en/docs/building-operators/golang/crds-scope.md

```diff
@@ -6,29 +6,61 @@ weight: 60
 
 ## Overview
 
-The CustomResourceDefinition (CRD) scope can also be changed for cluster-scoped operators so that there is only a single 
-instance (for a given name) of the Custom Resource (CR) to manage across the cluster.
+Custom Resource Definitions (CRDs) contain a scope field that determines whether the resulting Custom Resource (CR)
+is cluster or namespace scoped. Operators can be build making use of this feature that then operator either on the entire cluster,
```

Comment:
Operators can be built using this feature to ...

Reply:
wow, i don't know what happened there. thanks for the catch.